essential/index.html

```diff
@@ -88,22 +88,22 @@
                         </div>
                         <div class="tile">
```

Comment:
```suggestion
                        </div>
                        <div class="tile">
                          <div class="tile is-parent">
                            <div class="tile is-child is-8"></div>
                          </div>
                        </div>
                        <div class="tile">
```
Put this one inbetween for some extra padding, which is also done on the row above